packages/metadata/src/types.ts

```diff
@@ -77,13 +77,16 @@ export interface MetadataMap<T> {
  */
 export interface DesignTimeMethodMetadata {
   /**
-   * Type of the method itself. It is `Function`
+   * Type of the method itself. It is `Function` for methods, `undefined` for the constructor.
    */
```

Comment:
See the unit tests for examples of `DesignTimeMethodMetadata` values.

https://github.com/strongloop/loopback-next/blob/2f90a9531384c3e99afc9618d609b85ea9791e64/packages/metadata/src/__tests__/unit/inspector.unit.ts#L645-L676